# Capstone Project Week 4 - Clustering and Analysis in Los Angeles County neighborhoods

#### Applied Data Science Capstone by IBM/Coursera

#### Gabriel Philot

### Table of Contents:
___

##### 1- Introduction

##### 2- Data
___

## 1-Introduction

### Interest of this Capstone Project

The idea of this project is to use one of the most popular unsupervised machine learning techniques, for creating an analysis of a city’s venues , based on the information from foursquare’s available data of each neighborhood. This project also serve to obtain some knowledge before opening primarily a restaurant in LA, and other applications that need information on how different and similar neighborhoods are by the symmetries of their characteristics of each neighborhood.

### 1.2-Business Problem

The objective of this capstone project is to determine the best location for opening a new restaurant of Bahian food in the city of Los Angeles. The methodology that will be used for this project brings some Data Science and machine learning methods for answering the following business question: Where is the best location for an entrepreneur to open his restaurant?

## 2-Data

### 2.1 Data to solve the problem

List of all neighborhoods in Los Angeles, CA, USA.

Latitude and Longitude from these neighborhoods.

All the venues of each neighborhoods.

### 2.2 Data Source¶

##### In this project the data came from two diferent souces

https://usc.data.socrata.com/dataset/Los-Angeles-Neighborhood-Map/r8qd-yxsr - All the neighborhoods from Los Angeles and the latitude, longitude from each one neighborhood.


https://developer.foursquare.com/ - The availble venue data for each neighborhood.

In [4]:
import pandas as pd

### 2.4 Picking the data

#### 2.4.1 LA: Los Angeles city Neighborhoods, with latitude and longitude coordinates

In [5]:

# read csv file
la = pd.read_csv(r'C:\Users\flavi\Documents\la_final.csv')

# sor
la.sort_values(by='Neighborhood', axis=0, inplace=True)

la = la.set_index('Neighborhood').reset_index()

la.shape , la.head()

((271, 3),
       Neighborhood   Latitude   Longitude
 0            Acton  34.497355 -118.169810
 1  Adams-Normandie  34.031462 -118.300208
 2     Agoura Hills  34.146737 -118.759885
 3       Agua Dulce  34.504927 -118.317104
 4         Alhambra  34.085539 -118.136512)

This Dataframe set is composed of 271 neighborhoods with the following properties: Name, Latitude, and Longitude

The next Data Step will be on connecting to the foursquare API and make use of a function like de getnearby venues (that we seen on the course ) for extracting the first 100 venues of each neighboordhood, with all that Data avaible for us we can continue our Capstone Project in the 5th week.

Thats all for this notebook,see ya!